src/test/resources/regressions/features/channels/channel-fail6.gobra

```diff
@@ -8,11 +8,24 @@ pred sendInvariant(v int) {
 }
 
 // contracts for built-in members might use other built-in members which also need to be generated (even when not mentioned in the program text)
-requires c.IsChannel(1)
+requires c.IsChannel() && c.BufferSize() == 1
```

Comment:
I do not see a problem in using `BufferSize` for now but in the long term, we should probably use `cap` instead(https://golang.org/pkg/builtin/#cap)

Reply:
I've created #202 to not forget about this

Reply:
Thanks :)